### PR TITLE
Fix image event

### PR DESCRIPTION
**Pull request by noahSerafin**
image now changes with different eventIds

### CODE REVIEW

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import ViewScheduleButton from "../../components/ViewScheduleButton/ViewSchedule
 import Layout from "../../components/Layout/Layout";
 import { useParams } from "react-router-dom";
 import { getEvent } from "../../assets/data/api";
-import placeHolderImg from "../../assets/images/image33.png";
+//import placeHolderImg from "../../assets/images/image33.png";
```
Comment (DanForder) [inline src/pages/Home/Home.jsx:9]: remove commented code

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const Home = () => {
 
   if (!event) return <h1>Loading...</h1>;
 
-  const { name, series, time, date, venue, location, content, theme } = event;
+  const { name, series, time, date, venue, location, content, theme, imageSrc } = event;
```
Comment (DanForder) [inline src/pages/Home/Home.jsx:21]: we need the image alt as well, not just image src

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (noahSerafin) [reply src/pages/Home/Home.jsx]: removed

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -29,7 +28,8 @@ const Home = () => {
           date={date}
           venue={venue}
           location={location}
-          content={placeHolderImg}
+          content={imageSrc}
+          altcontent={imageAlt}
```
Comment (DanForder) [inline src/pages/Home/Home.jsx:32]: typoe : should be altContent

**Review by DanForder — CHANGES_REQUESTED** [review]

**Review by DanForder — APPROVED** [review]

**Pull request merged**